test/utils.ts

```diff
@@ -109,13 +109,27 @@ export async function blobToImage(blob: Blob): Promise<HTMLImageElement> {
 /**
  * Given a target length, and the Uint 8 RGBA channels for a pixel, create a
  * block of pixel values for testing sampled swatches of canvas data.
+ *
+ * @param pixelCount The number of pixels to produce.
+ * @param fillColor Array of four color channel values.
+ * @param convertOpacity Whether the fourth channel needs opacity-to-alpha
+ * conversion (multiply by 256).
  */
 export function filledColorArray(
     pixelCount: number,
-    fillColor: [number, number, number, number]): Uint8ClampedArray {
+    fillColor: number[],
+    convertOpacity = false,
```

Comment:
Minor enhancement to make it easier to create filled color arrays for pixel-wise testing.